lib/livebook_web/live/session_live/bin_component.ex

```diff
@@ -101,13 +111,50 @@ defmodule LivebookWeb.SessionLive.BinComponent do
                 </button>
               </div>
             <% end %>
+            <%= if @search != "" and @matching_entries == [] do %>
```

Comment:
There is some additional space in this case, because we render it within the `min-h-` list container. Since there are no matching entries to show, we can move this outside. Maybe a `cond`?

```heex
<%= cond do %>
  <% @bin_entries == [] -> %>
    nothing there

  <% @matching_entries == [] -> %>
    nothing found

  <% true -> %>
    entries list
<% end %>
```